src/components/generic/Modal/Modal.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {Modal, ModalBody, ModalHeader} from '@momentum-ui/react';
+import {v4 as uuidv4} from 'uuid';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../../constants';
+
+/**
+ * Modal component
+ *
+ * @param {object} props Data passed to the component
+ * @param {string} props.className  Custom CSS class to apply
+ * @param {string[]} props.children List of children
+ * @param {string} props.headerLabel Title of the Modal
+ * @param {Function} props.onHide Callback when the Modal is closed
+ * @param {boolean} props.show Show or hide the Modal
+ * @param {boolean} props.showCloseButton Show or hide the Modal close button
+ *
+ * @returns {object} JSX of the element
+ */
+export default function ModalComponent({
+  className,
+  children,
+  headerLabel,
+  onHide,
+  show,
+  showCloseButton,
+  ...props
+}) {
+  const mainClasses = {
+    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-modal`]: true,
+    [className]: !!className,
+  };
+  const customId = `${WEBEX_COMPONENTS_CLASS_PREFIX}-${uuidv4()}`;
```

Comment:
let's not introduce a new dependency. Just use `Math.random()` instead of uuid.

Reply:
I just saw that there was already an older version of the uuid installed so I can use that without installing anything new